app/views/schools/on_boarding/progresses/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<%- self.page_title = 'On-boarding Progress' -%>
+<%= govuk_back_link javascript: true %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <%= @current_school.name %> Profile
+    </h1>
+
+    <h2>Application Incomplete</h2>
```

Comment:
I think we can remove this H2 tbh @ethax-ross. It doesn't add much and takes up space. The relevant context, how many sections are complete, is below it and hidden as a result.